expression/builtin_arithmetic_vec.go

```diff
@@ -515,11 +515,11 @@ func (b *builtinArithmeticModRealSig) vecEvalReal(input *chunk.Chunk, result *ch
 	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
 		return err
 	}
+	result.MergeNulls(buf)
 	x := result.Float64s()
 	y := buf.Float64s()
 	for i := 0; i < n; i++ {
 		if buf.IsNull(i) {
```

Comment:
can we use `result.IsNull(i)` here?

Reply:
Tried before, could not work. There are cases where `buf.IsNull(i) == false && result.IsNull(i) == true`, leading to early termination, as a result, test(`builtin_arithmetic_vec_test.go:183`) alerts fewer wanrings detected on vec version.

![image](https://user-images.githubusercontent.com/34013484/105336367-1ff23380-5c14-11eb-871f-4902f38f851c.png)

Reply:
ok